.gitignore

```
@@ -83,3 +83,6 @@ prime/
 *.snap-build
 *_source.tar.bz2
 .DS_Store
+
+# Make evidence files
+/.make_evidence/*
```

```suggestion
/.make_evidence
```

may as well ignore the directory as a whole.